CMakeLists.txt

```diff
@@ -81,11 +82,15 @@ install(CODE "file(DOWNLOAD ${TITLE_SEQUENCE_URL} \$ENV{DESTDIR}${CMAKE_INSTALL_
 install(CODE "execute_process(COMMAND \"${CMAKE_COMMAND}\" -E chdir \$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/share/${PROJECT}/title/ \"${CMAKE_COMMAND}\" -E tar xvf title-sequences.zip)")
 install(CODE "file(REMOVE \$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/share/${PROJECT}/title/title-sequences.zip)")
 if (WIN32)
-    install(TARGETS "libopenrct2" RUNTIME DESTINATION bin)
-else()
-    install(TARGETS "libopenrct2" LIBRARY DESTINATION lib)
-endif()
-install(TARGETS "openrct2" RUNTIME DESTINATION bin)
+    install(TARGETS "libopenrct2" RUNTIME DESTINATION "bin")
+else ()
+    if (PORTABLE)
+        install(TARGETS "libopenrct2" LIBRARY DESTINATION "bin")
+    else ()
+        install(TARGETS "libopenrct2" LIBRARY DESTINATION "usr/local/lib")
```

Comment:
The DESTINATION here should just be "lib". cmake already adds the prefix, which may not be `/usr/local` - e.g. it's `/usr` in the Arch package.

Currently this results in `libopenrct2.so` being installed in `/usr/usr/local/lib`.